source/Octopus.Tentacle.Client/Scripts/UnsafeStartAttemptException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Octopus.Tentacle.Client.Scripts
+{
+    public class UnsafeStartAttemptException : Exception
```

Comment:
If this is used we should document what it is about.

Reply:
Done